test-harness/README.md

```diff
@@ -7,7 +7,8 @@
 
 ## Running the suite
 
-Run `yarn test.harness` from your terminal
+- Add a line to your `/etc/hosts` to map the httpbin hostname to the current remote service ip: `echo $(dig +short httpbin.org | head -n 1) httpbin >> /etc/host`
```

Comment:
```suggestion
- Add a line to your `/etc/hosts` to map the httpbin hostname to the current remote service ip: `echo $(dig +short httpbin.org | head -n 1) httpbin >> /etc/hosts`
```
A random thought: isn't it possible to force docker container to have a `httpbin.org` hostname instead of default `httpbin`? That would remove the need of user being forced to manually set things here.

-- edit --
Some manipulations here would do the trick: https://github.com/stoplightio/prism/blob/master/.circleci/config.yml#L59

I can investigate it later if you don't find time.

Reply:
You might be right. I'll try this out.

Reply:
It seems like you are right! I've just removed it.